packages/loot-core/src/server/main.ts

```diff
@@ -1892,13 +1904,103 @@ handlers['delete-budget'] = async function ({ id, cloudFileId }) {
 
   // If a local file exists, you can delete it by passing its local id
   if (id) {
+    // loading and then closing the budget is a hack to be able to delete
+    // the budget file if it hasn't been opened yet.  This needs a better
+    // way, but works for now.
+    await loadBudget(id);
```

Comment:
Pretty sure this wont work on slow devices - I remember fixing a bug for this. The problem is that some actions are farmed out to a worker process when you load the budget, closing it while those actions are happening causes a hard crash which can wipe out important files because they are being written to while the hard crash happens. 

The problem shows up on large budget files or on a slow harddrive while using Electron.

Can you find a better way to do this?

Reply:
Changed from loading and then closing the budget to just opening and closing the database.  This is still a hack to populate that part of the Absurd filesystem, but doesn't load all the other parts of the budget. 
Is this a better way?

Reply:
Seems to work. I agree it's still a bit hacky but I'm happier with this approach than the other way.